packages/tiler-sharp/src/index.ts

```diff
@@ -144,7 +144,12 @@ export class TileMakerSharp implements TileMaker {
 
     if (resize) {
       const resizeOptions = { fit: Sharp.fit.cover, kernel: resize.scaleX > 1 ? resizeKernel.in : resizeKernel.out };
-      sharp.resize(resize.width, resize.height, resizeOptions);
+      if (resize.scale <= 0.1251) {
```

Comment:
I think we need to link to the ticket for this, and maybe also expand on why the `.1` is used and why `0.125` (aka 1/8th)

Reply:
we can do this later.